src/encoded/static/components/item-pages/WorkflowRunTracingView.js

```diff
@@ -152,12 +152,8 @@ export class FileViewGraphSection extends WorkflowGraphSection {
             'key'       : 'graph-section',
             'disabled'  : !Array.isArray(steps) || steps.length === 0,
             'content'   : (
-                <FileViewGraphSection
-                    {...parentProps}
-                    {..._.pick(parentState, 'steps', 'mounted', 'allRuns')}
-                    key={"graph-for-" + context.uuid}
-                    onToggleAllRuns={onToggleAllRuns}
-                    loading={loadingGraphSteps} />
+                <FileViewGraphSection {...parentProps} {..._.pick(parentState, 'steps', 'mounted', 'allRuns')} width={width}
```

Comment:
Width is passed in as a prop now. (Continuation of comment/edit from ExperimentSetView.js.)